networkx/readwrite/mermaid.py

```diff
@@ -0,0 +1,238 @@
+"""
+****************
+Mermaid diagrams
+****************
+Read and write NetworkX graphs in Mermaid format.
+
+Mermaid is a collection of text-based diagrams that are well suited
+to be integrated with markdown and get visually rendered by JavaScript.
+
+While Mermaid has plenty of diagram formats [1], here in NetworkX
+we focus on flowcharts [2]
+
+[1] https://mermaid.js.org/intro/#diagram-types
+
+[2] https://mermaid.js.org/syntax/flowchart.html
+
+Format
+------
```

Comment:
I would prefer that this header be removed or at least not appear so prominently. Looking at the built docs (see the "details" of the CI test named "document artifact").
```suggestion
```